markdown/developers/sync.md

```diff
@@ -71,7 +71,7 @@ You will not need to touch it.
 ### Pull the changes to your fork
 
 On the command line, go to the directory where you have checked out your fork of the pipeline repository.
-Add the nf-core fork as a _git remote_ called `upstream`:
+Add the nf-core fork as a *git remote* called `upstream`:
```

Comment:
Oof, might be good if we choose the same one to use in all files 🤦🏻 

(maybe in another PR "one day" as we want a quick merge)